microcosm_pubsub/producer.py

```diff
@@ -81,6 +81,7 @@ def create_message(self, media_type, dct, uri=None, opaque_data=None, **kwargs)
         message = self.pubsub_message_schema_registry.find(media_type).encode(
             dct,
             opaque_data=opaque_data,
+            media_type=media_type,
```

Comment:
I don't know how this was working before, but without this, it just defaults to `application/json`.

Reply:
Well, it defaulting now because the field's value on `missing` is the `DEFAULT_MEDIA_TYPE`, which is `application/json`. I'm still not entirely sure why the de/serialize functions aren't being called (they should be on load/dump), but it doesn't hurt to explicitly specify the media type when creating the message (we find the message by media_type, so it's a bit it should just work out the same).